web/vtadmin/src/components/routes/workflows/WorkflowActions.tsx

```diff
@@ -22,7 +22,7 @@ const WorkflowActions: React.FC<WorkflowActionsProps> = ({ refetchWorkflows, key
 
     return (
         <div className="w-min inline-block">
-            <Dropdown dropdownButton={Icons.info} position="bottom-right">
```

Comment:
We don't need `position="bottom-right"`, as we want to render it on "bottom-left". But, with the dropdown fix we don't need to mention a position as by default dropdown menu renders on "bottom-left"